frontend/src/parseConfig.ts

```diff
@@ -20,3 +20,54 @@ window.Parse.enableLocalDatastore()
 Parse.CoreManager.setStorageController(Parse.IndexedDB)
 
 
+// import { SecureObject } from '/@common/models/base/SecureObject'
```

Comment:
also this?

Reply:
This is the way i originally wanted to implement it, but it's not working. I'll have to see with `Parse-JS` to see whats going on.

Even if we set our `ObjectController`, it's not called by the SDK

Reply:
ah, ok